README.md

```diff
@@ -111,123 +72,83 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Comming Soon 🎈]()
+
+> Presentation
+
+- [Loom Video](https://www.loom.com/share/3b36bd4c3b1e46b989b9692f013e7aec)
```

Comment:
- [ ] Nice video presentation, but you did not show a piece of code you are proud of. kindly include that in your video

Reply:
Thanks for the feedback